recipes/libuv/all/conanfile.py

```diff
@@ -0,0 +1,55 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class libuvConan(ConanFile):
+    name = "libuv"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libuv.org"
+    description = "A multi-platform support library with a focus on asynchronous I/O"
+    topics = ("libuv", "asynchronous", "io", "networking", "multi-platform", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+
+    exports_sources = ["patches/*"]
+
+    @property
+    def _source_subfolder_name(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder=self._source_subfolder_name)
```

Comment:
You need to add the cmake wrapper in order to configure options and settings correctly.

Reply:
cmake wrapper? Could you give a recipe as an example?

Reply:
https://github.com/conan-io/conan-center-index/blob/bc433494e8bcaa0b7348e5dc1001fefacc21168f/recipes/pcre2/all/CMakeLists.txt

Reply:
Done: bcdbdb2bfa2c5ebec4c316a8088f4843a2648177

That helped with `Windows x86_64, Debug, Visual Studio 14, MTd` failure, thank you. It was quite unexpected that Conan works with `CMAKE_MSVC_RUNTIME_LIBRARY` by the agency of `conan_basic_setup` unlike it does in the case with `BUILD_SHARED_LIBS` for example.